src/SprykerEco/Yves/Computop/Mapper/Init/PostPlace/DirectDebitMapper.php

```diff
@@ -0,0 +1,141 @@
+        );
+
+        $decryptedValues = $this->computopService->getEncryptedArray(
+            $this->getDataSubArray($computopPaymentTransfer),
+            $this->config->getBlowfishPassword()
+        );
+
+        $length = $decryptedValues[ComputopApiConfig::LENGTH];
+        $data = $decryptedValues[ComputopApiConfig::DATA];
+
+        $computopPaymentTransfer->setData($data);
+        $computopPaymentTransfer->setLen($length);
+        $computopPaymentTransfer->setUrl(
+            $this->getUrlToComputop(
+                $computopPaymentTransfer->getMerchantId(),
+                $data,
+                $length
+            )
+        );
+
+        return $computopPaymentTransfer;
+    }
+
+    /**
+     * @param \Spryker\Shared\Kernel\Transfer\TransferInterface|\Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return \Generated\Shared\Transfer\ComputopDirectDebitPaymentTransfer
+     */
+    protected function createTransferWithUnencryptedValues(TransferInterface $quoteTransfer)
+    {
+        $computopPaymentTransfer = new ComputopDirectDebitPaymentTransfer();
+
+        $computopPaymentTransfer->setCapture(ComputopSharedConfig::CAPTURE_MANUAL_TYPE);
+        $computopPaymentTransfer->setTransId($this->generateTransId($quoteTransfer));
+        $computopPaymentTransfer->setMandateId($computopPaymentTransfer->getTransId());
+        $computopPaymentTransfer->setUrlSuccess(
+            $this->getAbsoluteUrl($this->application->path(ComputopControllerProvider::DIRECT_DEBIT_SUCCESS))
+        );
+        $computopPaymentTransfer->setOrderDesc(
+            $this->computopService->getTestModeDescriptionValue($quoteTransfer->getItems()->getArrayCopy())
+        );
+
+        return $computopPaymentTransfer;
+    }
+
+    /**
+     * @param \Spryker\Shared\Kernel\Transfer\TransferInterface $cardPaymentTransfer
+     *
+     * @return array
+     */
+    protected function getDataSubArray(TransferInterface $cardPaymentTransfer)
+    {
+        /** @var \Generated\Shared\Transfer\ComputopDirectDebitPaymentTransfer $cardPaymentTransfer */
+        $dataSubArray[ComputopApiConfig::TRANS_ID] = $cardPaymentTransfer->getTransId();
+        $dataSubArray[ComputopApiConfig::AMOUNT] = $cardPaymentTransfer->getAmount();
+        $dataSubArray[ComputopApiConfig::CURRENCY] = $cardPaymentTransfer->getCurrency();
+        $dataSubArray[ComputopApiConfig::URL_SUCCESS] = $cardPaymentTransfer->getUrlSuccess();
+        $dataSubArray[ComputopApiConfig::URL_FAILURE] = $cardPaymentTransfer->getUrlFailure();
+        $dataSubArray[ComputopApiConfig::CAPTURE] = $cardPaymentTransfer->getCapture();
+        $dataSubArray[ComputopApiConfig::RESPONSE] = $cardPaymentTransfer->getResponse();
+        $dataSubArray[ComputopApiConfig::MAC] = $cardPaymentTransfer->getMac();
+        $dataSubArray[ComputopApiConfig::ORDER_DESC] = $cardPaymentTransfer->getOrderDesc();
+        $dataSubArray[ComputopApiConfig::ETI_ID] = ComputopConfig::ETI_ID;
```

Comment:
Can we move all config values into one place ?
`ComputopConfig::ETI_ID` and `ComputopSharedConfig::CAPTURE_MANUAL_TYPE`